app/repositories/job_repo.py

```
@@ -0,0 +1,129 @@
+        service = CompositeService(session)
+        service.compute_and_save_composite(voice_id)
+        logger.log_step("완료", category="async")
+        
+        # release lock (keep done flags)
+        row.locked = 0
+        session.commit()
+        
+        # voice_composite 생성 완료 → 연결된 CARE 사용자에게 알림 발송
+        try:
+            _send_composite_completion_notification(session, voice_id)
+        except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)